jumpscale/servers/threebot/threebot.py

```diff
@@ -54,7 +54,7 @@ def default_config(self):
                     "name": bottle_server.get("name"),
                     "host": bottle_server.get("host"),
                     "port": bottle_server.get("port"),
-                    "path_url": j.sals.fs.join_paths(self.package.base_url, bottle_server.get("path_url").lstrip("/")),
```

Comment:
we need the lstrip 
all paths passed to path.join (except the first one) shouldn't starts with `/`